src/core/builders/create-form-validator.spec.ts

```diff
@@ -1095,3 +1096,219 @@ describe("createFormValidator", () => {
     expect(validation).toEqual([{ field: Schema.string, error: null }]);
   });
 });
+
+describe("debounced validation", () => {
+  const Schema = createFormSchema(
+    fields => ({
+      string: fields.string(),
+      number: fields.number(),
+    }),
+    error => error<"REQUIRED" | "TOO_SHORT" | "INVALID_VALUE">()
+  );
+  const createFormValidatorImpl = (
+    ...args: Parameters<typeof createFormValidator>
+  ) => impl(createFormValidator(...args));
+
+  it("should debounce when single-rule debounced validator called >1 times", async () => {
+    const stringRequiredValidator = jest.fn((x: string) =>
+      x ? null : "REQUIRED"
+    );
+
+    const { validate } = createFormValidatorImpl(Schema, validate => [
+      validate({
+        field: Schema.string,
+        rules: () => [stringRequiredValidator],
+        debounce: 1000,
```

Comment:
the spec works nicely but using real debounce timers means it runs much longer than other specs
<img width="589" alt="Screenshot 2021-02-22 at 00 32 11" src="https://user-images.githubusercontent.com/12655670/108642493-af099a00-74a5-11eb-89ab-a2e9efa6b4ce.png">

maybe try https://jestjs.io/docs/en/timer-mocks (didn't really work last time I tried it) or at least decrease debounce values

Reply:
decreased debounced values
```
Test Suites: 28 passed, 28 total
Tests:       496 passed, 496 total
Snapshots:   0 total
Time:        2.558 s, estimated 3 s
```